pulp_rpm/app/depsolving.py

```diff
@@ -830,12 +810,246 @@ def run_solver_jobs(jobs):
                 unit_install_job = self._pool.Job(flags, solvable.id)
                 install_jobs.append(unit_install_job)
 
-        # Depsolve using the list of unit install jobs, add them to the results
-        solvables_copied = run_solver_jobs(install_jobs)
-        result_solvables.update(solvables_copied)
+        # Take a list of jobs, get a solution, return the set of solvables that needed to
+        # be installed.
+        solver = self._pool.Solver()
+        solver.set_flag(solv.Solver.SOLVER_FLAG_FOCUS_INSTALLED, 1)
+
+        raw_problems = solver.solve(install_jobs)
+        # The solver is simply ignoring the problems encountered and proceeds associating
+        # any new solvables/units. This might be reported back to the user one day over
+        # the REST API. For now, log only "real" dependency issues (typically some variant
+        # of "can't find the package"
+        dependency_warnings = self._build_warnings(raw_problems)
+        if dependency_warnings:
+            logger.warning(
+                "Encountered problems solving dependencies, "
+                "copy may be incomplete: {}".format(", ".join(dependency_warnings))
+            )
+
+        transaction = solver.transaction()
+        if settings.SOLVER_DEBUG_LOGS:
```

Comment:
I know we discussed whether this was a settings-thing or an API-parameter Thing. I don't remember where we left that discussion w/ iballou tho.

Reply:
If we're only writing a couple of Kb to `/var/tmp/pulp/` then I think it's safe to just leave this on by default all the time. And in that case there's no point making it an API parameter.

Even though I left the debug dump code it's disabled and there's no way for a user to enable it.  It's just not that useful except *maybe* for developers.

Reply:
Fair enough. Might be worth seeing if we can get feedback from the folks asking for this as well.

RE debug-dump - yeah, good call, I think it *will* be useful on occasion to us, but it's def way too much to be useful to anyone who isn't hip-deep in libsolv already.